test/unittests/test_utils.py

```diff
@@ -1250,3 +1250,55 @@ def test_ping_node(mock_run):
         utils.ping_node("node_unreachable")
     assert str(err.value) == 'host "node_unreachable" is unreachable: error data'
     mock_run.assert_called_once_with("ping -c 1 node_unreachable")
+
+
+def test_is_quorate():
+    assert utils.is_quorate(3, 2) is True
+    assert utils.is_quorate(3, 1) is False
```

Comment:
Not necessary, but a test like `utils.is_quorate(4, 2) is False` would also make sense.